docs/application/native/guides/app-management/app-controls.md

```diff
@@ -808,7 +812,50 @@ The following common application controls are available:
 - [Voice Recorder](common-appcontrols.md#voice-recorder)
 - [VPN Service](common-appcontrols.md#vpn-service)
 
-## Related Information
+<a name="URI"></a>
+## Application control URI
+
+> [!NOTE]
+>
+> Application Control URI is supported since Tizen 5.5.
```

Comment:
@kilig Thanks for your patience, request you to kindly remove > , like this 

```suggestion 
> [!NOTE]
> Application Control URI is supported since Tizen 5.5.